test/ProgressOnderwijsUtils.Tests/Data/ParameterizedSqlTest.cs

```diff
@@ -355,6 +355,15 @@ public void ParameterizedSqlDoesNotUseLiteralsEnumsMarked_IEnumShouldBeParameter
 
     [TestNotLiteral]
     enum ExampleNonLiteralEnum { SomeValue = 1, }
+
+    [Fact]
+    public void GenerateUniqueQueryAlias_is_in_fact_unique_within_query()
```

Comment:
Ik denk dat je hier netto de heel test-set ui Query_wrapper_avoids_name_clash wil meenemen, want o.a. die determinisme test is relevant voor query-plan pollution, en het bevat ook asserts voor wat gebeurt bij een name-clash